go.mod

```
@@ -91,6 +90,7 @@ require (
 	github.com/mailgun/ttlmap v0.0.0-20170619185759-c1c17f74874f
 	github.com/mattn/go-sqlite3 v1.14.16
 	github.com/mdlayher/netlink v1.7.1
+	github.com/microsoft/go-mssqldb v0.0.0-00010101000000-000000000000
```

```suggestion
	github.com/microsoft/go-mssqldb v0.0.0-00010101000000-000000000000 // replaced
```

So it's easier to spot in case something wrongly tries to update it here (like Dependabot).